src/components/Link/tests/Link.test.tsx

```diff
@@ -123,4 +123,30 @@ describe('<Link />', () => {
       });
     });
   });
+
+  describe('accessibilityLabel', () => {
+    it('passes prop to the button url is not provided', () => {
+      const mockAccessibilityLabel = 'mock accessibility label';
+      const link = mountWithApp(
+        <Link accessibilityLabel={mockAccessibilityLabel} />,
+      );
+
+      expect(link).toContainReactComponent('button', {
+        'aria-label': mockAccessibilityLabel,
+      });
+    });
+
+    it('passes prop', () => {
+      const mockAccessibilityLabel = 'mock accessibility label';
+
+      const link = mountWithApp<LinkProps>(
```

Comment:
Why is this needed? I thought TS would interpolate the props just fine here

Reply:
For some reason is not getting interpolated correctly: 
![image](https://user-images.githubusercontent.com/4788817/101517256-ca044d80-393d-11eb-88e9-77c399fbcd0a.png)